### PR TITLE
Show channels for only active server in autocomplete

### LINKED ISSUE #1381: Autocomplete for channels should maybe only show channels for current server
Current behaviour:
Autocomplete shows the list of channels across all servers that I'm connected to when I type `#`

Is it possible that only the channels for the current server to be displayed (since referencing someone to a channel in _another_ server is not usual I think).

How does this suggestion sound? I might also be able to help out if needed.

### PR DESCRIPTION
**Pull request by milindl**
Fix #1381.

I hope this is correct. It seems to work for me.

### CODE REVIEW

File: client/js/autocompletion.js [hunk 1]
```diff
@@ -282,7 +282,7 @@ function completeCommands(word) {
 function completeChans(word) {
 	const words = [];
 
-	sidebar.find(".chan")
+	sidebar.find(".chan.active")
```
Comment (xPaw) [inline client/js/autocompletion.js]: This will only find the active channel, not network. You have to go up to the parent.

Comment (milindl) [reply client/js/autocompletion.js]: I've now fixed it, and the commits are squashed into one as the Contributing guidelines say.

Comment (xPaw) [reply client/js/autocompletion.js]: Please change spaces to tabs on the lines that have been added. We use tabs in our codebase.

Comment (milindl) [reply client/js/autocompletion.js]: I am sorry I missed that, I have also fixed that now.

**Pull request merged**